fabric-api-catalog/build.gradle

```
@@ -0,0 +1,57 @@
+plugins {
+	id 'version-catalog'
+}
+
+version = rootProject.version
+
+publishing.publications {
+	register('mavenJava', MavenPublication) {
+		from(components['versionCatalog'])
```

```suggestion
		from components.versionCatalog
```
More conventional in the Groovy DSL